src/DocScan/DocScanClient.php

```diff
@@ -53,9 +54,7 @@ public function __construct(
         $pemFile = PemFile::resolveFromString($pem);
 
         // Set API URL from environment variable.
-        if (getenv(Constants::ENV_DOC_SCAN_API_URL) !== false && !isset($options[Config::API_URL])) {
-            $options[Config::API_URL] = getenv(Constants::ENV_DOC_SCAN_API_URL);
-        }
+        $options[Config::API_URL] = $options[Config::API_URL] ?? Env::get(Constants::ENV_DOC_SCAN_API_URL);
```

Comment:
In PHP 7.4 this can become:
```suggestion
        $options[Config::API_URL] ??= Env::get(Constants::ENV_DOC_SCAN_API_URL);
```